### ISSUE TITLE
PML1: possible typo in 10.2.3.1

### ISSUE THREAD

**Issue by AndreaPi** [probml/pml-book#194]
![image](https://user-images.githubusercontent.com/8233615/130634780-eaadc53d-6f7d-4ba8-8452-c2faf7a9d940.png)

I think that, formally, the connection with cross-entropy would be made more precise if eq. 10.9 (and necessarily all the preceding equations) included a $\frac 1 N$ factor, i.e.

\begin{align}
     =& -\frac 1 N \sum_{n=1}^N [y_n\log\mu_n+(1-y_n)(\log(1-\mu_n)] \\
     =& -\frac 1 N \sum_{n=1}^N\mathbb{H}(y_n,\mu_n)
\end{align}

The reason is that if we want to intepret the negative log-likelihood as the cross-entropy between the _empirical label distribution_ and the label distribution generated by the model, then the empirical label distribution corresponds to putting a probability atom of $\frac 1 N$ on the observed training data and zero mass everywhere else. This is coherent with section **6.1.5 Perplexity**
![image](https://user-images.githubusercontent.com/8233615/130638939-ee2ed38d-f680-40a6-8902-dcfada3efd07.png)

and section **6.2.5 KL divergence and MLE**, where the connection between minimizing the _average_ negative log-likelihood, and minimizing the cross-entropy between the empirical distribution and the predicted distribution is made very clear:

![image](https://user-images.githubusercontent.com/8233615/130638653-e7fd3672-5910-48a4-8737-f421f0b61f01.png)
![image](https://user-images.githubusercontent.com/8233615/130638719-c16c1e94-7106-4087-aeb2-5b076fb5e3b0.png)

Now obviously a constant multplying factor doesn't make any difference for a minimization procedure, and the fitted model will be exactly the same, whether we include the factor $\frac 1 N$ in front of the NLL or not. However, given the excellent effort of the book in being as self-contained as reasonably possible, without requiring prior knowledge of information theory, probability, etc., I think a little bit more clarity/consistency would be gained if the $\frac 1 N$ were to be included in section 10.2.3.1, same as in the preceding sections 6.1.5 and 6.2.5.

**Comment by murphyk** [comment]
Yes I agree, this is a good suggestion, thanks.
